Podcast/SubscriptionCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SubscriptionCollectionViewCell.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscriptionCollectionViewCell: RecommendedSeriesCollectionViewCell {
```

Comment:
why is this subclassing recommendedseriescollectionviewcell? isn't this supposed to be completely unrelated to any other cell?